syntax = "proto3";

service FileSearchService {
  rpc Search(SearchRequest) returns (SearchResponse);
}

message SearchRequest {
  string file_name = 1;
  repeated bytes data = 2;
}

message SearchResponse {
  repeated string ids = 1;
}